### PR TITLE
adding client ID to retrieve

### PR DESCRIPTION
**Pull request by jimmyburgess91**
I've modified the retrieve function signature to replace the filter param with an options param.  This options param allows you to specify the clientID (business unit) on retrieve requests.  This pertains to the issue at https://github.com/ExactTarget/Fuel-Node-SOAP/issues/45.

### CODE REVIEW

**Comment by aydrian** [comment]
Please make the discussed offline changes and make sure jshint runs successfully.

**Comment by vernak2539** [comment]
Also remove the '?' in the banner. not sure if you talked about that

File: lib/fuel-soap.js [hunk 1]
```diff
@@ -163,44 +163,68 @@ FuelSoap.prototype.create = function (type, props, options, callback) {
...
-				'xmlns': 'http://exacttarget.com/wsdl/partnerAPI'
-			},
-			'RetrieveRequest': {
-				'ObjectType': type,
-				'Properties': props
+	if(clientIDs) {
+		var body = {
+			'RetrieveRequestMsg': {
+				'$': {
+					'xmlns': 'http://exacttarget.com/wsdl/partnerAPI'
```
Comment (vernak2539) [inline lib/fuel-soap.js]: should the xmlns be a config value based on which env your trying to use?

**Comment by aydrian** [comment]
It took me a while to find the '?'. Perhaps @jimmyburgess91 was confused whether to update the year since it's now 2015. We can just keep it 2014, right?

**Comment by vernak2539** [comment]
I would update it to 2015

**Comment by aydrian** [comment]
Any further issues before I merge this in? @vernak2539 @kellyjandrews

**Comment by vernak2539** [comment]
with this change, why weren't any of the tests updated? I would think some test would need something changed, or is this part not tested?

**Comment by kellyjandrews** [comment]
+1

**Comment by jimmyburgess91** [comment]
working on updating the test cases

**Comment by vernak2539** [comment]
:+1:. @aydrian when those are done, it should be fine to merge

**Comment by vernak2539** [comment]
@jimmyburgess91 sorry it took so long to get this merged. Thanks for the work!

**Pull request merged**